packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -416,11 +419,37 @@ const tabPanel = style({
 }, getAllowedOverrides({height: true}));
 
 export function TabPanel(props: TabPanelProps) {
+  let {showTabs} = useContext(CollapseContext);
+  let {selectedKey} = useContext(InternalTabsContext);
+  if (showTabs) {
+    return (
+      <AriaTabPanel
+        {...props}
+        style={props.UNSAFE_style}
+        className={props.UNSAFE_className + tabPanel(null, props.styles)} />
+    );
+  }
+
+  if (props.id !== selectedKey) {
+    return null;
+  }
+
+  return <CollapsedTabPanel {...props} />;
+}
+
+function CollapsedTabPanel(props: TabPanelProps) {
+  let {UNSAFE_style, UNSAFE_className = '', ...otherProps} = props;
+  let {menuId} = useContext(CollapseContext);
+  let ref = useRef(null);
+  let tabIndex = useHasTabbableChild(ref) ? undefined : 0;
```

Comment:
I added two inputs to the Example story and after collapsing, it still has a tabIndex of 0

Probably just need to pass the ref to the Group